### PR TITLE
Fix #1341: Increase the width of edit details dialog box

### LINKED ISSUE #1341: Increase the width of edit details dialog box
**Observed Behaviour :**
dialog_box
The dialog box looks small in the screenshot.

**Expected Behaviour:** Increase the width of the dialog box suitably and paste the screenshot in the PR.

### PR DESCRIPTION
**Pull request by S2606**
IMPORTANT NOTES (please read, then delete):

* The PR title should start with "Fix #bugnum: " (if applicable), followed by a clear one-line present-tense summary of the changes introduced in the PR. For example: "Fix #bugnum: Introduce the first version of the collection editor.".

* Please make sure to mention "#bugnum" somewhere in the description of the PR. This enables Github to link the PR to the corresponding bug.

Please also make sure to follow the [style rules](https://github.com/Cloud-CV/EvalAI/blob/master/CONTRIBUTING.md#style-rules).

### CODE REVIEW

**Comment by Obtin** [comment]
@S2606  edit the descrription please. Thank you :-)

**Comment by aka-jain** [comment]
@S2606 Please attach the screenshot of the final result and also edit your description.

File: frontend/src/views/web/challenge/edit-challenge/edit-challenge-submission-guidelines.html [hunk 1]
```diff
@@ -1,22 +1,22 @@
-<section class="text-center pd-20 width-500">
+<section class="text-center pd-20 width-700">
```
Comment (aka-jain) [inline frontend/src/views/web/challenge/edit-challenge/edit-challenge-submission-guidelines.html]: Also, where is this class defined? I think, currently we have only 'width-500' class presdent. You have to create a new one for 'width-700'. @S2606

Comment (S2606) [reply frontend/src/views/web/challenge/edit-challenge/edit-challenge-submission-guidelines.html]: @aka-jain ok thank you!!

**Comment by aka-jain** [comment]
@S2606 you working on this?

**Comment by RishabhJain2018** [comment]
Since @S2606 has been inactive. So, I am closing this PR.The issue is now open to everyone!

**Pull request closed without merging**